src/test/java/com/redhat/cloud/policies/app/OapiTest.java

```diff
@@ -64,5 +69,8 @@ public void validateOpenApi() throws Exception {
  		// User config is private, so don't show it
  		Assert.assertFalse(paths.containsKey("/user-config"));
  		Assert.assertFalse(schemas.containsKey("SettingsValues"));
+
+		InputStream in = url.openStream();
```

Comment:
I think it would be good to document this.
Someone new to the project may not understand why this exists in a test.